CMakeLists.txt

```diff
@@ -313,6 +313,15 @@ if(IS_LINUX AND IS_AMD64)
         SOURCE ${AMALGAM_LIB_ONLY_SOURCE} ${COMMON_SOURCE_THREADS} IDE_FOLDER "OtherBuildTargets")
 endif()
 
+# WASM single-threaded debug target (amalgam-st-debug)
+if(IS_WASM)
```

Comment:
Should we remove the wasm64-debug target? Looks like this will always build the debug variant along side the release variant. Might be cleaner to only build debug when its the target?

Reply:
Yeah, I was initially debating a refactor to the CI/CD build script to simply execute release + debug builds for WASM (instead of shoehorning in a WASM debug target). The main draw towards this for me was that it'd plop both WASMs in the same build artifact with different naming conventions (-st and -st-debug), so that would save a bit of a lift in the CI/CD. But perhaps that was the lazy way. @fulpm if you're fine with this and want me to remove the "proper" wasm-debug target, I will, but if you think it'd be best to simply use that and remove this shoehorned one, that's fine too. Just let me know.

Reply:
That is fine, lets just remove the `-debug` build presets since they wont be needed.